crates/hyperswitch_domain_models/src/merchant_connector_account.rs

```diff
@@ -567,34 +550,20 @@ common_utils::create_list_wrapper!(
         pub fn get_connector_and_supporting_payment_method_type_for_session_call(
             &self,
         ) -> Vec<(&MerchantConnectorAccount, common_enums::PaymentMethodType)> {
+            // This vector is created to work around lifetimes
+            let ref_vector = Vec::default();
+
             let connector_and_supporting_payment_method_type = self.iter().flat_map(|connector_account| {
```

Comment:
Follow the proper naming conventions for payment_method_type and payment_method_sub_type in the below implementation